src/jdk.compiler/share/classes/com/sun/tools/javac/code/Source.java

```diff
@@ -237,6 +237,7 @@ public enum Feature {
         UNCONDITIONAL_PATTERN_IN_INSTANCEOF(JDK21, Fragments.FeatureUnconditionalPatternsInInstanceof, DiagKind.PLURAL),
         RECORD_PATTERNS(JDK21, Fragments.FeatureDeconstructionPatterns, DiagKind.PLURAL),
         WARN_ON_ILLEGAL_UTF8(MIN, JDK21),
+        UNNAMED_VARIABLES(JDK21, Fragments.FeatureUnnamedVariables, DiagKind.PLURAL),
```

Comment:
While the JEP unifies unnamed variables and unnamed patterns, would it make sense, at the code level, to split between the two?